tests/analysis/interfaces/test_zsl.py

```diff
@@ -33,6 +34,7 @@ def setUp(self):
             self.get_structure("TiO2"), symprec=0.1
         ).get_conventional_standard_structure()
 
+    @pytest.mark.skip(reason="TODO: to be fixed right now.")
```

Comment:
I would try to fix this in this PR (hopefully), interestingly, after un-commenting `hiphive` (which depends on `numba`), this unit test would fail (meaning this test passes if `numba` is not installed).

Code snippet to recreate this:
```python

import numpy as np
from pytest import approx

from pymatgen.analysis.interfaces.zsl import fast_norm


assert fast_norm(np.array([3, 2, 1])) == approx(3.74165738)

```

Reply:
Never mind, see docstring: https://github.com/materialsproject/pymatgen/blob/3fee7d58fdc963b8fb8faf76ce92b68f7ccd0bf9/src/pymatgen/analysis/interfaces/zsl.py#L258-L266